spec/rubocop/cop/rspec/capybara/match_style_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::Capybara::MatchStyle, :config do
+  it 'registers an offense when using `assert_style`' do
+    expect_offense(<<~RUBY)
+      expect do
+        page.find(:css, '#first').assert_style(display: 'block')
+                                  ^^^^^^^^^^^^ Use `#good_method` instead of `#bad_method`.
+      end.not_to raise_error
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect do
+        page.find(:css, '#first').assert_matches_style(display: 'block')
+      end.not_to raise_error
+    RUBY
+  end
+
+  it 'registers an offense when using `has_style?`' do
+    expect_offense(<<~RUBY)
+      expect(page.find(:css, 'first')
+        .has_style?(display: 'block')).to be true
+         ^^^^^^^^^^ Use `#good_method` instead of `#bad_method`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect(page.find(:css, 'first')
+        .matches_style?(display: 'block')).to be true
+    RUBY
+  end
+
+  it 'registers an offense when using `have_style`' do
+    expect_offense(<<~RUBY)
+      expect(page).to have_style(display: 'block')
+                      ^^^^^^^^^^ Use `#good_method` instead of `#bad_method`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect(page).to match_style(display: 'block')
+    RUBY
+  end
+
+  it 'does not register an offense when using `assert_matches_style`' do
+    expect_no_offenses(<<~RUBY)
+      expect do
+        page.find(:css, '#first').assert_matches_style(display: 'block')
+      end.not_to raise_error
```

Comment:
Is `not_to raise_error` necessary? Is it used this way? (here and above)

Reply:
One last leftover

Reply:
Oops, sorry. I remove it.